CondTools/SiPixel/data/SiPixelTemplateDBObject_0T_phase1_BoR3_v1/generror_summary_zp0310.out

```diff
@@ -0,0 +1,1437 @@
+Dot1_150x100_dj0260a@-375V,0.0T@90d,263K,rh=1.10/0.7,1.3/1.3 
```

Comment:
Hi @tvami, these huge files do not belong to cmssw. Please remove them and squash your commit history in order not to clog the repo.